Packages/Sites/Neos.NeosIo/Resources/Private/Templates/NodeTypes/Reference.List.html

```diff
@@ -1,3 +1,25 @@
-<div {attributes -> f:format.raw()}>
-	{references -> f:format.raw()}
+{namespace io=Neos\NeosIo\ViewHelpers}<div {attributes -> f:format.raw()}>
+	<io:simpleForm actionUri="#reference-filter" class="form form--inline" method="get" id="reference-filter">
```

Comment:
I added the ViewHelper in the meantime
The line break I omitted intentionally to avoid empty lines when the `{namespace ..}` declaration is removed. But we can ofc re-add it for better readability